tools/fix_indent.py

```diff
@@ -130,49 +131,49 @@ def even_quotes(string):
         # Check that this line ends with ampersand
         if not empty_line:
             this_line_ends_ampersand = False
-            if re.search('&\s*$', to_output):
+            if re.search(r'\&\s*$', to_output):
                 this_line_ends_ampersand = True
 
             # This line starts with 'end', so the next line should be unindented
             if not previous_line_ends_ampersand:
-                if re.match('^\s*end\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*else\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*contains\s*', to_output, flags=re.IGNORECASE):
+                if re.match(r'^\s*end\s*', to_output, flags=re.IGNORECASE) \
+                  or re.match(r'^\s*else\s*', to_output, flags=re.IGNORECASE) \
+                  or re.match(r'^\s*contains\s*', to_output, flags=re.IGNORECASE):
                     indent = indent - 1
 
                 # Handle the fact that each case of a 'select' structure doesn't
                 # end in an 'end', so the 'end select' needs to go back twice
-                if re.match('^\s*end select\s*', to_output, flags=re.IGNORECASE):
+                if re.match(r'^\s*end select\s*', to_output, flags=re.IGNORECASE):
                     indent = indent - 1
 
                 # Match 'if-then-else', 'do', 'subroutine', 'function', 'module', 'contains',
                 # 'program', 'interface', 'select', 'case', 'type' (definition, not instantiation)
                 # to set the next line indent higher
-                if re.search('\s*IF.+THEN', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*else\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*do\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*subroutine\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*function\s*', to_output, flags=re.IGNORECASE) \
-                  or (re.match('^\s*module\s*', to_output, flags=re.IGNORECASE) \
-                      and not re.match('^\s*module procedure \s*', to_output, flags=re.IGNORECASE)) \
-                  or re.match('^\s*contains\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*program\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*interface\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*abstract interface\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*pure function\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*select\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*case\s*', to_output, flags=re.IGNORECASE) \
-                  or re.match('^\s*type\s+', to_output, flags=re.IGNORECASE):
+                if re.search(r'\s*if.+then', to_output, flags=re.IGNORECASE) \
```

Comment:
I changed this because the `fix_style.py` script changes IF-THEN-ELSE to lowercase, but it shouldn't make a difference given the flag

Reply:
Makes sense